ubuntu-core-20-amd64-intel-iotg.json

```diff
@@ -0,0 +1,38 @@
+{
+    "type": "model",
+    "series": "16",
+    "authority-id": "canonical",
+    "brand-id": "canonical",
+    "revision": "1",
+    "model": "ubuntu-core-20-amd64-intel-iotg",
+    "architecture": "amd64",
+    "timestamp": "2021-06-29T19:37:37+00:00",
+    "base": "core20",
+    "grade": "signed",
+    "snaps": [
+        {
+            "name": "pc",
+            "type": "gadget",
+            "default-channel": "20/stable",
+            "id": "UqFziVZDHLSyO3TqSWgNBoAdHbLI4dAH"
+        },
+        {
+            "name": "intel-kernel",
+            "type": "kernel",
+            "default-channel": "20/stable",
+            "id": "eKpzfdgTkbItdivsTeTx1EnDTpfZ1Ua4"
```

Comment:
Note that at the moment intel-kernel is only published up to 20/beta. I have tested the kernel snap with my self-built model assertions so it is working correctly. I guess we should promote it, all the way to 20/stable.

Reply:
My thought was to get them all signed at once even if we wouldn't be building them right away.